docs/developers/emp-deployment.md

```diff
@@ -79,16 +79,20 @@ yarn
-- `excessTokenBeneficiary` should be set to the UMA contract store by default. `0x41AF40Eb92Bec4dD8DA77103597838b3dBBD3B6f` for Kovan and `0x54f44eA3D2e7aA0ac089c4d8F7C93C27844057BF` for Mainnet.
+- `expirationTimestamp`: This should be the exact Unix timestamp that your contract should expire at. It is recommended that you set this to 10:00 pm UTC on your desired expiry day. [Here](https://www.unixtimestamp.com/) is a helpful tool for finding a timestamp. 
+- `collateralAddress`: Lists the token address of an approved collateral currency on the network you are trying to deploy to. As an example, deploying a contract collateralized by WETH to Kovan should use the Kovan WETH address `0xd0a1e359811322d97991e03f863a0c30c2cf029c`.
+- `priceFeedIdentifier`: Matches the exact name of the approved price identifier converted to bytes32 format. Example for USDETH: `0x555344455448`.
+- `syntheticName`: The plaintext synthetic token name. Example: `Synthetic Test USDETH Dec 2024`.
+- `collateralRequirement`: The [collateralization requirement ratio](https://docs.umaproject.org/synthetic-tokens/glossary#collateralization-requirement) scaled to 18 decimals. Example for a 1.5 ratio: `{ rawValue: '1500000000000000000' }`. Please note that it is potentially dangerous to set this requirement too low. Only in rare cases for stable financial products should this ever be set below 1.2, and never below 1.05.
+- `collateralRequirement`: Should always be at or higher than `1.25`.
+- `minSponsorTokens`: Should be targeted to approximately $100 of value at your synthetic token's expected price. If your synthetic token is expected to be worth $1, `minSponsorTokens` should be `{ rawValue: toWei("100") }`.
+- `withdrawalLiveness`: The length of the [withdrawal liveness period](https://docs.umaproject.org/synthetic-tokens/glossary#withdrawal-liveness-period) in seconds. Example for 2 hours: `7200`.
+- `liquidationLiveness`: The length of the [liquidation liveness period](https://docs.umaproject.org/synthetic-tokens/glossary#liquidation-liveness-period) in seconds. Example for 2 hours: `7200`.
+- `disputerDisputeRewardPercentage`: The percentage size of the dispute reward paid to the disputer scaled to 18 decimals. Example for 20%: `{ rawValue: '200000000000000000' }`.
+- `disputeBondPercentage`: The percentage size of the [dispute bond](../synthetic-tokens/expiring-synthetic-tokens#liquidation-and-dispute) converted to 18 decimals. Example for 10%: `{ rawValue: '100000000000000000' }`.
+- `sponsorDisputeRewardPercentage`: The percentage size of the [dispute reward](../synthetic-tokens/expiring-synthetic-tokens#liquidation-and-dispute) paid to the position sponsor scaled to 18 decimals. Example for 5%: `{ rawValue: '50000000000000000' }`.
+- `financialProductLibraryAddress`: The address of a contract that can transform a DVM price received by an EMP. For situations where this does not apply, this should be set to `0x0000000000000000000000000000000000000000`.
```

Comment:
Looks good.

As part of the Post Deployment section on this page, there is a link to our outdated autogenerated docs: https://docs-dot-uma-protocol.appspot.com/uma/contracts/ExpiringMultiParty.html.

Can you replace this with the generated docs from here https://docs-git-doc-updates-uma.vercel.app/contracts/Contracts ? 

There are also a bunch of links to these in other places. Example would be the sidebar item `Contract Documentation` under Developer Reference. Can we replace all of these?

Reply:
How are you finding these? I searched and updated the two above and then another in docs/developers/optimistic-oracle-integration.md but not seeing any others when searching.

Reply:
Actually I was mistaken. Those three were all that I saw